src/CVAE/index.js

```diff
@@ -28,13 +29,11 @@ class Cvae {
     this.ready = false;
     this.model = {};
     this.latentDim = tf.randomUniform([1, 16]);
-    this.modelPath = modelPath;
-    this.modelPathPrefix = '';
 
-    this.jsonLoader().then(val => {
-      this.modelPathPrefix = this.modelPath.split('manifest.json')[0]
-      this.ready = callCallback(this.loadCVAEModel(this.modelPathPrefix+val.model), callback);
-      this.labels = val.labels;
+    const [modelPathPrefix] = modelPath.split('manifest.json');
+    axios.get(modelPath).then(({ data }) => {
+      this.ready = callCallback(this.loadCVAEModel(modelPathPrefix + data.model), callback);
```

Comment:
I kept the structure here the same as it was before in terms of "fetch then callCallback".  But now that I'm looking over it, the API call really should be moved **inside** of `callCallback` to get proper error handling.  Probably something for a future PR.  I've done a bunch of IIFEs in other places for things like this.